recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ source:
     - 0013-javareconf-do-not-fail-on-compile-fail.patch
 
 build:
-  number: 2
+  number: 0
```

Comment:
@bgruening Shouldn't this be bumped to build number 3 if the plan is to stick with R 3.4.1?

Reply:
Oh, yes good catch.